src/systems/camera.js

```diff
@@ -39,6 +39,7 @@ module.exports.System = registerSystem('camera', {
       defaultCameraEl.setAttribute('camera', {active: true, userHeight: DEFAULT_USER_HEIGHT});
       defaultCameraEl.setAttribute('wasd-controls', '');
       defaultCameraEl.setAttribute('look-controls', '');
+      defaultCameraEl.setAttribute('aframe-injected', '');
```

Comment:
Could perhaps define this string as a constant somewhere. Maybe `constants/index.js`